<?xml version='1.0'?>
<launch>

<node name="stereo2depth" pkg="stereo_depth" type="stereo_depth">
	<remap from="left/image" to="/camera1/infra1/image_rect_raw"/>
	<remap from="right/image" to="/camera1/infra2/image_rect_raw"/>
	<remap from="left/camera_info" to="/camera1/infra1/camera_info"/>
	<remap from="right/camera_info" to="/camera1/infra2/camera_info"/>

	<param name="baseline" value="0.05"/>
	<param name="scale" value="1.0"/>
</node>

</launch>
